tests/test_spahm_b.py

```diff
@@ -47,8 +60,8 @@ def test_water_same_basis():
 def test_ecp():
     path = os.path.dirname(os.path.realpath(__file__))
     xyz_in = path+'/data/I2.xyz'
-    mols = utils.load_mols([xyz_in], [0], [None], 'minao', ecp='def2-svp')
-    dms = utils.mols_guess(mols, [xyz_in], 'LB', spin=[None])
```

Comment:
this was actually weird ! because even specifying `spin=None` the func would still returned merged alpha-beta representations (which is the `X_true` we loaded and evaluated) !

But it made me realized that it's a bit dangerous now too, because if you only use the `bond()` function with a single dm (for all electrons) and you don't change the parameters `omods=[None]` (default is `[alpha, beta]`) then you get an error ! (... I think ...)